### DISCUSSION TITLE
ClockDivider to DualTM weirdness

### DISCUSSION THREAD

**Discussion by riverbasin** [djphazer/O_C-Phazerville#49] [Q&A]
Hello,

I am using Pamela's Pro Workout to send a clock to ClockDivder on the left hemisphere and the sending the divider/multiplied signal to DualTM digital 1 on the right. As long as ClockDivider is set to divide, I can use Pam's as master clock and play control. However, If I set it multiply, it seems an internal clock takes over and the DualTM runs whether I am sending a signal from Pam's or not. Is this expected behavior?  How can I set things up so I can use my master clock to control the O_C?

**Comment by djphazer** [comment]
Yes, unfortunately that's how Clock Multiplication works in that applet. It's much simpler to do clock division by skipping incoming clocks. Multiplication requires predicting the tempo and generating extra pulses. Automatically stopping becomes tricky...

**Reply by djphazer** [reply]
I'm going to eat my words on this one. Automatic stop is just a matter of counting.
Stay tuned for a fix...

**Reply by riverbasin** [reply]
Hey Phazer - just an FYI. I just tried flashing both o_C-phazerville-v1.7.0_T40-0747cac.hex and o_C-phazerville-v1.7.0_T41-0747cac.hex.  Teensy.app threw a warning about the file size being too large (see attachment).  It allows me to flash anyways, which yielded a blank screen and no functionality.  I reverted to 1.6.99 with no problems.

In any event, keep up the good work.  Thanks for your effort!

<img width="270" alt="Screenshot 2024-01-30 at 8 45 41 AM" src="https://github.com/djphazer/O_C-Phazerville/assets/1075713/8011ecff-fcda-46ce-adcd-c58558bdb30d">

**Reply by riverbasin** [reply]
I forgot to mention this a Teensy 3.2

**Reply by djphazer** [reply]
Teensy 4.0 and 4.1 builds obviously don't work on Teensy 3.2! Try one of the _phz builds...

**Reply by riverbasin** [reply]
Right, I overlooked that! ;-)  I tried o_C-phazerville-v1.7.0_T41-0747cac.hex and good to go!  Thanks
